include/RAJA/operators.hxx

```diff
@@ -245,11 +245,11 @@ template <typename T>
 RAJA_HOST_DEVICE constexpr T min()
 {
   return 0;
-};
+}
 template <>
 RAJA_HOST_DEVICE constexpr int16_t min<int16_t>()
 {
-  return 0xFFFF;
+  return -32768;
```

Comment:
This demonstrates two ways to do it:

    #include <iostream>
    #include <cstdint>
    #include <limits>
    
    int main(int argc, char **argv) {
      std::cout << "Max int16 via macro: " << INT16_MIN
                << "\nMax int16 via numeric_limits: "
                << std::numeric_limits<int16_t>::min()
                << "\n";
    }